### ISSUE TITLE
Create a page/section on the website for Svelte Sirens

### ISSUE THREAD

**Issue by kevmodrome** [svelte-society/sveltesociety.dev#140]
We need this! 🥳

**Comment by brittneypostma** [comment]
Sweet, I might do this instead of working on the website this weekend.

**Comment by brittneypostma** [comment]
Kind of curious where everyone thinks this should live though. Could just be listed under Societies around the World on the Events page or maybe just a blurb that links to the page in the Home page paragraph?

**Comment by kevmodrome** [comment]
Yeah I'm not sure, could be a top level category as well 🤔 I'm open to pretty much everything here I think.

**Comment by stephane-vanraes** [comment]
A top level category would disrupt the 'balance' it has now `A B C Logo D E F` which feels nice, if we add the sirens there as well one side will have more elements which is always a bit weird.

On the home page you get into the same problem if you make it one of the 6 already there 🙂 so you could mention it in the texts above or below (where Svelte Summit is mentioned).

I would for sure put it on top of the `Societies around the world` though, before the Europe section.
